test/gurka/test_avoids.cc

```diff
@@ -214,6 +216,44 @@ TEST_P(AvoidTest, TestAvoid2Polygons) {
   };
 }
 
+TEST_P(AvoidTest, TestAvoidShortcuts) {
+  auto node_b = avoid_map.nodes.at("B");
+  auto node_c = avoid_map.nodes.at("C");
+  auto node_e = avoid_map.nodes.at("E");
+  auto dx = std::abs(node_c.lng() - node_b.lng());
+  auto dy = std::abs(node_b.lat() - node_e.lat());
+
+  // create a small polygon on the shortcut: should fail if shortcuts are also avoided
+  //                x-x
+  //      A------B--|-|-C--F
+  //                x-x
```

Comment:
this does not actually work right now: it would succeed even if the shortcut was not in the avoid edges set. Any advice for building the map to make sure there's a shortcut I intersect with the tiny polygon?